tests/queries/0_stateless/02919_storage_fuzzjson.sql

```diff
@@ -42,3 +42,15 @@ CREATE TABLE 02919_test_table_reuse_args(str String) ENGINE = FuzzJSON(
 SELECT count() FROM (SELECT * FROM 02919_test_table_reuse_args LIMIT 100);
 
 DROP TABLE IF EXISTS 02919_test_table_reuse_args;
+
+--
+DROP TABLE IF EXISTS 02919_test_table_invalid_col_type;
+CREATE TABLE 02919_test_table_invalid_col_type
+(
+   str Nullable(Int64)
+)
+ENGINE = FuzzJSON('{"pet":"rat"}', NULL);
```

Comment:
Shouldn't we forbid creating such storages in the first place and throw an exception during the CREATE query?